composer/dag_test_utils/setup.py

```diff
@@ -16,12 +16,12 @@
 from setuptools import setup
 
 setup(
-    name="dag_test_utils",
+    name="cloud_composer_dag_test_utils",
     version="0.0.1",
-    url="git@github.com:GoogleCloudPlatform/python-docs-samples.git#egg=dag_test_utils&subdirectory=composer/dag_test_utils",
-    author="Google Cloud Platform",
-    description="Utility used to unit test example Apache Airflow DAGs",
+    url="https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/composer/dag_test_utils",
+    author="Google LLC",
+    description="Internal utility used to unit test example Apache Airflow DAGs for Google Cloud Composer",
```

Comment:
nit: isn't this now, by publishing, not so internal? I don't know what other word to use though to best indicate.

Reply:
Perhaps you could also add "this is not an officially supported Google product." (wording from https://www.npmjs.com/package/@google/repo)  to emphasize this isn't for external folks?

Reply:
Lol yes - that was my conundrum as well. I think the combo of both suggestions is appropriate - remove the word internal, but emphasize that it is not officially supported.